src/tv2-common/getSegment.ts

```diff
@@ -125,6 +126,10 @@ export async function getSegmentBase<ShowStyleConfig extends TV2ShowStyleConfig>
 		segment.isHidden = false
 	}
 
+	segment.definesShowStyleVariant = iNewsStory.cues.some((cue) =>
+		cue ? cue[0].toLowerCase().includes('showstylevariant') : false
```

Comment:
On the offchance that someone accidentally didn't place the showstyleVariant keyword on line one of the cue, maybe we should check all lines of the cue?

Also, the cue is `Sofie=Showstylevariant`, not just `showstyleVariant`. And it would be nice with a constant instead of a magic string :)

Reply:
It now checks all cues and uses a constant instead :-)